modules/hostap/Kconfig

```diff
@@ -292,6 +292,10 @@ config WIFI_NM_WPA_SUPPLICANT_DPP
 	select MBEDTLS_X509_CSR_WRITE_C
 	select MBEDTLS_X509_CSR_PARSE_C
 
+config WIFI_NM_WPA_SUPPLICAT_11AX
```

Comment:
Typo in the name `s/WIFI_NM_WPA_SUPPLICAT_11AX/WIFI_NM_WPA_SUPPLICANT_11AX/`

Reply:
Updated